python/ray/_raylet.pyx

```diff
@@ -1630,8 +1637,24 @@ cdef class CoreWorker:
                 c_actor_id,
                 ray_function,
                 args_vector, CTaskOptions(name, num_returns, c_resources))
-
-            return VectorToObjectRefs(return_refs)
+            if return_refs.has_value():
+                return VectorToObjectRefs(return_refs.value())
+            else:
+                actor = self.get_actor_handle(actor_id)
+                actor_handle = (CCoreWorkerProcess.GetCoreWorker()
+                                .GetActorHandle(c_actor_id))
+                raise BackPressureError("Cannot submit the actor task {} "
```

Comment:
cc @ericl on sign off for error message

Reply:
How about this?

```
raise PendingCallsLimitExceeded(
   "The task could not be submitted because more than {} ""
   "tasks are queued on the actor. This limit can be adjusted "
   "with the `max_pending_calls` actor option.")
```